### DISCUSSION TITLE
Clicking notify-send box hangs the system

### DISCUSSION THREAD

**Discussion by gh-supersudo** [i3/i3#6325] [Q&A]
I’m using Manjaro i3. Whenever there’s a notify-send message, the system hangs for a couple of seconds when I click the box.

Even running a simple command then clicking on the box lags the system.

`notify-send "Foo Bar"`

What could be the cause of this?

**Comment by okraits** [comment]
Do you have a notification daemon installed?

**Reply by gh-supersudo** [reply]
I have dunst installed.

```
$ dunst -v
Dunst - A customizable and lightweight notification-daemon 1.11.0 (2024-04-15)
```

**Reply by okraits** [reply]
Is it running?

**Reply by gh-supersudo** [reply]
It is.

~~I noticed that when I disable and reenable dunst again, it doesn't lag anymore when the box has been clicked.~~

I noticed that this only happens when I'm using the mouse keys.
